pom.xml

```diff
@@ -3,15 +3,14 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.55</version>
+        <version>4.53</version>
```

Comment:
Why downgrading here?

Reply:
Sorry, was looking at an intermediate change or something, it showed downgrading from 4.55 to 4.53